hack/run-e2e-tests.sh

```diff
@@ -13,8 +13,8 @@ docker build . \
     -t "kudobuilder/controller:$VERSION"
 
 # Generate the kudo.yaml that is used to install KUDO while running e2e-test
-./bin/kubectl-kudo init --dry-run --output yaml \
-    | sed -E -e "/image:/ s/:[[:alnum:]]+/:$VERSION/" -e '/imagePullPolicy/ s/Always/Never/' \
+./bin/kubectl-kudo init --dry-run --output yaml --kudo-image kudobuilder/controller:$VERSION \
```

Comment:
This is great, I didn't know about `kudo-image`. Now that I've _actually_ looked into the command documentation, how about using `version` instead? It'll only override the controller version.

Reply:
```suggestion
./bin/kubectl-kudo init --dry-run --output yaml --version $VERSION \
```
(not tested)

Reply:
Sadly, that always prefixes the version with letter `v`.

Reply:
But since we specify the name of the image on line 13 anyway, I think fully specifying it here as well will be more robust.

Reply:
Got it, makes sense.